app/javascript/components/ProductTable.vue

```diff
@@ -1,5 +1,5 @@
 <template lang="html">
-  <b-container slot="row-details" slot-scope="row">
```

Comment:
It looked like this was redundant, but I actually do not know. Can anyone with more knowledge of this test this out?

Reply:
I've added two commits dealing with v-slot usage. You can take a look at them, if you'd like